AnimationSystem/constants.h

```diff
@@ -0,0 +1,844 @@
+enum BoneIndex {

```

Comment:
Can we use IOTA for this so there are no explicit numbers?

Reply:
OK, I'll change.

Reply:
Used existing bone indexes infos, instead of explicit numbers and iota.
https://github.com/webaverse/app-wasm/pull/75/commits/7c6b01bbd80c001a5f46711ecfde36ebaa2000ae
https://github.com/webaverse/app-wasm/pull/75/commits/8d30b38e277531c626023eebcf041c3dfa3fd5a6